index.bs

```diff
@@ -5144,7 +5144,7 @@ The [=type name=] of the
 
 The {{unrestricted float}} type is a floating point numeric
 type that corresponds to the set of all possible single-precision 32 bit
-IEEE 754 floating point numbers, finite and non-finite.  [[!IEEE-754]]
+IEEE 754 floating point numbers, finite, non-finite, and special "not a number" values (NaNs). [[!IEEE-754]]
```

Comment:
Do we mean to allow all NaN representations, versus the ES single special NaN value?